website/docs/r/virtual_machine.html.markdown

```diff
@@ -1,58 +1,236 @@
+machines, creating templates to clone from, or migrating from previous versions
+of this resource.
+
+### Disks
+
+The `vsphere_virtual_machine` resource currently only supports standard
+VMDK-backed virtual disks - it does not support other special kinds of disk
+devices like RDM disks.
+
+Disks are managed by exact name supplied to the `name` attribute of a [`disk`
+sub-resource](#disk-options). This is required - the resource does not support
+automatic naming.
+
+Virtual disks can be SCSI disks only. The SCSI controllers managed by Terraform
+can vary, depending on the value supplied to
+[`scsi_controller_count`](#scsi_controller_count). This also dictates the
+controllers that are checked when looking for disks during a cloning process.
+By default, this value is `1`, meaning that you can have up to 15 disks
+configured on a virtual machine. These are all configured with the controller
+type defined by the [`scsi_type`](#scsi_type) setting. If you are cloning from
+a template, devices will be added or re-configured as necessary.
+
+When cloning from a template, you must specify disks of either the same or
+greater size than the disks in source template when creating a traditional
+clone, or exactly the same size when cloning from snapshot (also known as a
+linked clone). For more details, see the section on [creating a virtual machine
+from a template](#creating-a-virtual-machine-from-a-template).
+
+A maximum of 60 virtual disks can be configured when the
+[`scsi_controller_count`](#scsi_controller_count) setting is configured to its
+maximum of `4` controllers. See the [disk options](#disk-options) section for
+more details.
+
+### Customization and network waiters
+
+Terraform waits during various parts of a virtual machine deployment to ensure
+that it is in a correct expected state before proceeding. These happen when a
+VM is created, or also when it's updated, depending on the waiter.
+
+Two waiters of note are:
+
+* **The customization waiter:** This waiter watches events in vSphere to
+  monitor when customization on a virtual machine completes during VM creation.
+  Depending on your vSphere or VM configuration it may be necessary to change
+  the timeout or turn it off. This can be controlled by the
+  [`timeout`](#timeout-1) setting in the [customization
+  settings](#virtual-machine-customization) block.
+* **The network waiter:** This waiter waits for a _routeable_ interface to show
+  up on a guest virtual machine close to the end of both VM creation and
+  update. This waiter is necessary to ensure that correct IP information gets
+  reported to the guest virtual machine, mainly to facilitate the availability
+  of a valid, routeable default IP address for any
+  [provisioners][tf-docs-provisioners]. This option can be managed or turned
+  off via the [`wait_for_guest_net_timeout`](#wait_for_guest_net_timeout)
+  top-level setting.
+
+[tf-docs-provisioners]: http://localhost:4567/docs/provisioners/index.html
```

Comment:
you'll want to remove `http://localhost:4567` here

Reply:
Yikes, that's a bug - thanks